blog/2024/10-26-jellyfin-release-10.10.0/index.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Jellyfin 10.10.0
+description: We are pleased to announce the latest stable release of Jellyfin, version 10.10.0!
+authors: joshuaboniface
+slug: jellyfin-release-10.10.0
+tags: [release]
+---
+
+We are pleased to announce the latest stable release of Jellyfin, version 10.10.0!
+
+This major release brings several new features, improvements, and bugfixes to improve your Jellyfin experience. With our faster release cadence between 10.9.0 and 10.10.0, this release should be far less daunting, so please read on for a quick peek at what's new and some important-to-know breaking changes!
+
+You may upgrade your Jellyfin instances at any time now. For those who were running Unstable builds for testing, we thank you immensely, and you may now switch back to the Stable repository and forcibly reinstall/repull the latest version. As always, **ensure you back up your Jellyfin data and configuration directories before upgrading**. With a major release, it's possible you will hit a bug and want to revert, and to do so, you will need to restore from a backup.
+
+Happy watching!
+
+\- Joshua
+
+{/* truncate */}
+
+## Breaking Changes
+
+* Most clients should continue to work as-is without any issues or any forced upgrades, though this may change in the future. The sole exception is Jellyfin4Kodi, which currently has issues due to point 5 below, though a fix is hopefully forthcoming shortly.
+
+* We have deprecated Raspberry Pi V4L2 hardware transcoding support, and are looking to deprecate any Raspberry Pi-specific functionality in general for releases past 10.10.0. Raspberry Pi has always been a tricky platform for us, as it's popular but extremely poor in performance, and this has caused a lot of people a lot of headaches. With the RPi5 entirely removing a hardware acceleration engine and the aging of the RPi4, we are taking this action now to ensure it's widely known.  If you are running Jellyfin on old RPi hardware, now is the time to start looking at an upgrade. See [PR #1148](https://github.com/jellyfin/jellyfin.org/pull/1148).
```

Comment:
might want to link the hardware selection guide for people looking for an upgrade? https://jellyfin.org/docs/general/administration/hardware-selection